src/app/medInria/medLogger.cpp

```diff
@@ -1,5 +1,5 @@
 #include "medLogger.h"
-
+#include <filesystem>
```

Comment:
```
medLogger.cpp:2:10: fatal error: filesystem: Aucun fichier ou dossier de ce type
 #include <filesystem>
```
:/, i tried also to remove my build repertory and compile from scratch

Reply:
(#include <experimental/filesystem> is found if needed)

Reply:
Hum, your compiler/stdc++ is too old.
Experimental is the old place.
Maybe we can found the good include with pragma.
After is very easy to use the good namespace by namespace fs =